airflow/ui/src/pages/Run/Header.tsx

```diff
@@ -74,6 +74,10 @@ export const Header = ({
       { label: "Start", value: <Time datetime={dagRun.start_date} /> },
       { label: "End", value: <Time datetime={dagRun.end_date} /> },
       { label: "Duration", value: `${getDuration(dagRun.start_date, dagRun.end_date)}s` },
+      {
+        label: "Dag Versions at time of run",
```

Comment:
```suggestion
        label: "Dag Version(s)",
```

Let's simplify this label

Reply:
I took what was on the figma, but glad to update.

Reply:
Done.